README.md

```diff
@@ -22,7 +22,7 @@ use:
 <dependency>
   <groupId>com.echobox</groupId>
   <artifactId>ebx-linkedin-sdk</artifactId>
-  <version>1.0.6</version>
+  <version>1.0.7</version>
```

Comment:
I don't think this will work. I can't see the 1.0.7 in [here](https://mvnrepository.com/artifact/com.echobox/ebx-linkedin-sdk) . If the work in progress version is 1.0.7 i think we should use the 1.0.6. Will we release to maven the new version ?